Training5/manageapp/src/redux/action/UserAction.js

```diff
@@ -0,0 +1,20 @@
+import { ACTION_NAMES } from "./ActionNames";
+
+export const LoginUser = (data) => {
```

Comment:
@Huynhkimphat phat
Action creator should be naming like as method eg: `loginUser`, `getAllUsersInfo`